docs/docs/building-in-the-enterprise.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Building With Gatsby in the Enterprise"
+---
+
+Some Gatsby advocates may be working inside “enterprise” software development organizations. Others may work at agencies which work closely with such organizations.
+
+If this describes you, you have unique challenges, but also opportunities, building with Gatsby at your company.
+
+This guide is designed to help you navigate the terrain in order to effectively advocate for Gatsby within your organization. 
+
+### Common characteristics of enterprises
+
+People mean different things when they say “enterprise”. Some mean 500-person companies. Other people mean 20,000-person companies. 
+
+When we say “enterprise” we mean large companies with established infrastructural setups -- in-house layers of code, frameworks, processes, and preferred vendors to handle concerns such as hosting, deployments, CDN, etc.
+
+Other common characteristics include specified procurement processes, standardization on particular technology stacks, changeover at eg 5-year intervals, and frequent security audits. Often, enterprises are heavily invested in maintaining and improving existing web properties (“brownfield”) as opposed to creating new ones (“greenfield”).
```

Comment:
> changeover at eg 5-year intervals

This is a little unclear to me. Is that supposed to be `e.g.`?

Reply:
Yep! Fixed.